include/seqan3/alignment/configuration/align_config_method.hpp

```diff
@@ -28,12 +29,60 @@ namespace seqan3::align_cfg
  *
  * \details
  *
- * The alignment algorithm can be categorised in different methods. For example, the
- * \ref seqan3::align_cfg::method_local "local" and the
- * \ref seqan3::align_cfg::method_global "global" alignment are two different methods, while the semi-global alignment
- * is a variation of the global alignment. This differentiation makes it possible to define a subset of configurations
- * that can work with a particular method. Since it is not possible to guess what the desired method for a user is,
- * there is no default and this configuration must always be provided for the alignment algorithm.
+ * There are several methods for sequence alignment. We distinguish between \ref seqan3::align_cfg::method_local "local"
+ * and \ref seqan3::align_cfg::method_global "global" alignments. The semi-global alignment is implemented as a
+ * variation of the global alignment.
+ *
+ * **Global Alignment**:\verbatim
+--T--CC-C-AGT--TATGT-CAGGGGACACG-A-GCATGCAGA-GAC
+  |  || |  ||  | | | |||    || | | |  | ||||   |
+AATTGCCGCC-GTCGT-T-TTCAG----CA-GTTATG-T-CAGAT--C\endverbatim
+ * Finding the optimal global alignment of two sequences is solved by the **Needleman-Wunsch algorithm**.
+ *
+ * **Local Alignment** (better suited to find conserved segment):
+ * \verbatim
+                  tccCAGTTATGTCAGgggacacgagcatgcagagac
+                     ||||||||||||
+aattgccgccgtcgttttcagCAGTTATGTCAGatc
+\endverbatim
+ * A \ref seqan3::align_cfg::method_local "local" alignment is effectively a global alignment of two partial sequences,
+ * that is, when two genes from different species are similar in short conserved regions and dissimilar in the remaining
```

Comment:
```suggestion
 * A \ref seqan3::align_cfg::method_local "local" alignment is effectively a global alignment of two partial sequences.
 * For example when two genes from different species are similar in short conserved regions and dissimilar in the remaining
```
Or is `that is` the same as `for example`? That is.. sounds to me as this is the only case.